tools/ide_plugins/pysa-vscode/package.json

```diff
@@ -38,5 +38,34 @@
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^3.5.1",
     "vscode": "^1.1.7"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+      "languages": [{
```

Comment:
Nit: The naming of the two languages seems to put `.pysa` files above `taint.config` files, since `.pysa` files are the ones that get put into the "pysa" language. What if we put these on the same level, perhaps naming one `pysa_models` and the other `pysa_config`, or something like that?

Reply:
I like that idea! I just made those changes and reflected them in the file and scope naming as well.